content/automate/ManageIQ/System/Notification/Email.class/__methods__/miq_provision_customize_body.rb

```diff
@@ -21,7 +21,7 @@ def build_body
               signature = @handle.object['signature']
               # VM Provisioned Email Body
               body = "Hello,"
-              body += "<br/><br/>Your request to provision a Virtual Machine was approved and completed on #{@time}"
+              body += "<br/><br/>Your request to provision a Virtual Machine was approved and completed on #{time}"
```

Comment:
Thanks @jeffmcutter, this looks good.  

Wondering if you would not mind doing a small cleanup related to this.

In the `def time` method below I noticed we are calling `to_s` on the result of `strftime` which already returns a string, so the `to_s` is not required.

See https://github.com/ManageIQ/manageiq-content/blob/aa694264a1067b614c63adcc4c802fc335589dbc/content/automate/ManageIQ/System/Notification/Email.class/__methods__/miq_provision_customize_body.rb#L56-L58

Let me know either way.  Thanks.